mappings/remind_2.1.yaml

```diff
@@ -22,8 +22,9 @@ native_regions:
   - SSA: REMIND 2.1|SSA
   - UKI: REMIND 2.1|UKI
   - USA: REMIND 2.1|USA
-  - EUR: EU27 & UK
-  - World
+  - EU27 & UK: REMIND 2.1|EUR
+  - NEU: REMIND 2.1|NEU
+  - World: REMIND 2.1|World
```

Comment:
The current implementation saves the "World" region as a common-region. The proposed change would save it (only) as a REMIND-native region, so it would not be possible to easily compare the global aggregates between models.

In principle, the model-specific identifier "<Model>|..." should only be used where we see a risk of confusion between different models. The definition of "World" should be pretty much identical across models, so no need to add the model-identifier.

```suggestion
```

Reply:
Following the same principle as "World", shouldn't we need to modify DEU and FRA mappings to have their values available on the model comparisons? They are single country results that are available directly from REMIND native regions.

Reply:
Well, I would treat all regions other than "World" like this:

 - include every model-native region as "<Model>|<Native Region>" in the database, i.e., if a model has 28 regions, there should be 28 entries in the region-definitions files and (at least) 28 entries in the native-region mapping file.
 - if a native-region and a common-region are identical, duplicate it by adding a common-region mapping, in your case
    ```yaml
   - Germany:
        - DEU
    ```
 - for uploading scenario data directly at a common-region level (for example because automated region-processing does not support a particular computation), the native-region-renaming feature can be used, i.e,, 
    ```yaml
   - EUR: EU27 & UK
    ```

Following this logic, a user (in the Scenario Explorer or working with an exported scenario ensemble data file) can immediately find all native regions from every model by taking all regions that start with the model-identifier.

There is a trade-off here between duplication of data and user-friendliness, and I would go for user-friendliness (except for World, where it is obvious that all global models report these values and there  are very little ambiguities about the definition).